### PR TITLE
updated ID syntax for wormbase

### LINKED ISSUE #628: fix wb links

### PR DESCRIPTION
**Pull request by nathandunn**
fixes #628

### CODE REVIEW

**Comment by vanaukenk** [comment]
@kltm 
Transgenes are a distinct entity from genes, so the URL would be different.  Here's an example:

https://wormbase.org/species/all/transgene/WBTransgene00001056

It looks like the SO id for Transgene is:  SO:0000902
http://www.sequenceontology.org/browser/current_svn/term/SO:0000902

Otherwise, this looks okay.

**Comment by kltm** [comment]
@vanaukenk Thank you so much for the input.

@nathandunn Could you change the URLs to reflect this?

**Comment by nathandunn** [comment]
Will do. 

Nathan

**Comment by nathandunn** [comment]
@kltm  Added transgene.  Ready to merge.

**Comment by kltm** [comment]
@vanaukenk Sorry to bother again, but could we borrow your eyes one more time?

**Comment by vanaukenk** [comment]
@kltm 
Sure, for which bit specifically?

**Comment by kltm** [comment]
@vanaukenk Sorry, I'm still catching up on email (i.e. the back and forth with Chris and Kevin). I think I'm nervous as this is such a big departure from what we had before. Honestly, I guess I was just looking for general hand holding from the WB side, but I think we're probably ready to move forward.

**Comment by kltm** [comment]
@nathandunn @cmungall As far as I can tell, this change is substantive enough from how things were originally done that is would cause breakage in AmiGO and the JS linker as they do not currently parse out types by regexp.

**Comment by nathandunn** [comment]
I promise to fix these when I can see straight.  

If these are fixed properly here they should work though, right? 

Nathan

**Comment by kltm** [comment]
I think there is still some discussion about what we should do with this change in the first place...

File: metadata/db-xrefs.yaml [hunk 1]
```diff
@@ -3233,22 +3233,34 @@
   entity_types:
     - type_name: gene
       type_id: SO:0000704
-      id_syntax: (WP:CE[0-9]{5})|(WB(Gene|Var|RNAi|Transgene)[0-9]{8})
-      url_syntax: http://www.wormbase.org/db/gene/gene?name=[example_id]
+      id_syntax: WBGene[0-9]{8})
```
[the remark this answers is not in the document]
Comment (nathandunn) [reply metadata/db-xrefs.yaml]: I think this is correct, unless you need parentheses around everything, but looking down the document, I don't see this as a requirement.

**Comment by pgaudet** [comment]
@kltm Has this been merged ?

**Comment by kltm** [comment]
@nathandunn @pgaudet I don't believe that will fix anything and would in fact cause things to start breaking. A good write-up of this issue can be found here: https://github.com/geneontology/go-site/issues/659#issuecomment-392196555

**Pull request closed without merging**